static/js/search/search_input.tsx

```diff
@@ -0,0 +1,285 @@
+              onClick={() => redirectAction(inputText, "")}
+              key={"search-input-result-0"}
+            >
+              {inputText}
+            </div>
+            {!_.isEmpty(results.placeResults) &&
+              results.placeResults.map((result, idx) => {
+                return (
+                  <div
+                    className={`search-input-result ${
+                      idx + 1 === hoveredIdx
+                        ? "search-input-result-highlighted"
+                        : ""
+                    }`}
+                    key={"search-input-result-" + result.dcid}
+                    onClick={() => redirectAction(result.name, result.dcid)}
+                  >
+                    {getHighlightedJSX(result.dcid, result.name, matches)}
+                  </div>
+                );
+              })}
+            {!_.isEmpty(results.svResults) &&
+              results.svResults.map((result, idx) => {
+                return (
+                  <div
+                    className={`search-input-result ${
+                      idx + 1 + results.placeResults.length === hoveredIdx
+                        ? "search-input-result-highlighted"
+                        : ""
+                    }`}
+                    onClick={() => redirectAction(result.name, "")}
+                    key={"search-input-result-" + result.dcid}
+                  >
+                    {getHighlightedJSX(result.dcid, result.name, matches)}
+                  </div>
+                );
+              })}
+          </div>
+        )}
+      </div>
+      <span
+        className="material-icons search-input-search-button"
+        onClick={() => redirectAction(inputText, "")}
+      >
+        search
+      </span>
+    </>
+  );
+
+  function onInputChanged(input: string): void {
+    setInputText(input);
+    if (input === "") {
+      setShowResults(false);
+      setResults({ placeResults: [], svResults: [] });
+      return;
+    }
+    setHoveredIdx(0);
+    setShowResults(true);
+    clearTimeout(delayTimer.current);
+    delayTimer.current = setTimeout(() => search(input), DELAY_MS);
+  }
+
+  function search(query: string): void {
+    latestQuery.current = query;
+    if (placeAutocompleteService.current) {
+      placeAutocompleteService.current.getPredictions(
+        { input: query, types: ["(regions)"] },
+        (predictions, status) =>
+          onAutocompleteCompleted(query, predictions, status)
+      );
+    } else {
+      getSvResultsPromise(query)
+        .then((results) => {
+          if (query !== latestQuery.current) {
+            return;
+          }
+          setResults({ placeResults: [], svResults: results });
+        })
+        .catch(() => {
+          if (query !== latestQuery.current) {
+            return;
+          }
+          setShowResults(false);
+          setResults({ placeResults: [], svResults: [] });
+        });
+    }
+  }
+
+  function onAutocompleteCompleted(
```

Comment:
nit: maybe clarify in the handler name that this is for PlaceAutocomplete

Reply:
done